src/main/java/org/sunbird/profile/controller/ProfileController.java

```diff
@@ -65,7 +65,14 @@ public ResponseEntity<?> parichayProfileUpdate(@RequestBody Map<String, Object>
 
 	@GetMapping("/user/v1/autocomplete/{searchTerm}")
 	public ResponseEntity<?> userAutoComplete(@PathVariable("searchTerm") String searchTerm) {
-		SBApiResponse response = profileService.userAutoComplete(searchTerm);
+		SBApiResponse response = profileService.userAutoComplete(searchTerm, null);
+		return new ResponseEntity<>(response, response.getResponseCode());
+	}
+
+	@GetMapping("/user/v1/mdo/autocomplete/{searchTerm}")
```

Comment:
instead of mdo - use "admin"

Reply:
The path has been changed from mdo to admin